mig-api/context.go

```diff
@@ -113,6 +114,7 @@ func initDB(orig_ctx Context) (ctx Context, err error) {
 		ctx.Channels.Log <- mig.Log{Desc: "leaving initDB()"}.Debug()
 	}()
 
+	fmt.Fprintf(os.Stdout, "Attempting to connect to postgresql database...")
```

Comment:
I thought it would be useful to add some sort of debug statement to print stating a connection is attempting to be made. Unfortunately the Ping function will take ~30 seconds to timeout, so the mig-api/scheduler could look like it's hanging.

Reply:
I believe at this point, we're unable to use "ctx.Channels.Log <- mig.Log"  to log, since the logger isn't setup yet?

Reply:
I tried using:
```
ctx.Channels.Log <- mig.Log{Desc: "Attempting to connect to db"}"
```
however, it didn't show up in the output?